compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -2001,25 +2001,21 @@ static bool evaluateIntComparison(TR_ComparisonTypes compareType, bool isUnsigne
       {
       case TR_cmpEQ:
          return left == right;
-         break;
       case TR_cmpNE:
          return left != right;
-         break;
       case TR_cmpLT:
          return isUnsignedCompare ? (uint64_t)left < (uint64_t)right : left < right;
-         break;
       case TR_cmpLE:
          return isUnsignedCompare ? (uint64_t)left <= (uint64_t)right : left <= right;
-         break;
       case TR_cmpGT:
          return isUnsignedCompare ? (uint64_t)left > (uint64_t)right : left > right;
-         break;
       case TR_cmpGE:
          return isUnsignedCompare ? (uint64_t)left >= (uint64_t)right : left >= right;
-         break;
       default:
          TR_ASSERT(false, "unhandled TR_ComparisonTypes enum value in simplifier");
```

Comment:
Rather than a default return false this should become a FATAL assert since the program will not behave correctly with the default return false.

Reply:
Done.